src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -409,6 +411,7 @@ export default {
 			inviterName: '',
 			inviterIsGuestOrAnonymous: false,
 			inPfp: false,
+			itemsInBasket: [],
```

Comment:
This along with the 2 above were causing issues b/c they weren't defined on data and consequently weren't reactive.